packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,15 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
+## Troubleshooting
 
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+### Custom Netlify redirect rules
+
+During compilation a required "catch all" redirect rule is automatically appended to your `_redirects` file. (If it doesn't exist yet, it will be created.) That means:
+
+- If you have a `from = "/*"` rule in `netlify.toml`, it will never match, as `_redirects` has a [higher priority](https://docs.netlify.com/routing/redirects/#rule-processing-order).
```

Comment:
If I'm reading those docs correctly, it's not just `/*` that won't match but any redirect defined in `netlify.toml`